src/gui/contacts/list.sh

```diff
@@ -0,0 +1,40 @@
+export tmpfile=$(mktemp)
+
+on_select() {
+    selected=$1
+    echo $selected > $tmpfile
+}
+export -f on_select
+
+gui_contacts_list(){
+    get_contacts "$@" | yad --title="EasyGnuPG | Contacts" \
+        --list \
+        --width=600 \
+        --height=450 \
+        --column="ID" \
+        --column="UID(s)" \
+        --button="Details":'bash -c "gui contacts_details $(head -n 1 $tmpfile)"' \
+        --button="Add Contact":'bash -c "gui contacts_add"' \
+        --button=gtk-quit \
+        --select-action='bash -c "on_select %s"'\
+        --dclick-action='bash -c "gui contacts_details $(head -n 1 $tmpfile)"'\
+        --no-rules-hint
```

Comment:
It seems to me that using checkboxes on the list might be nice. However I will have to try it first and see how it looks.